### PR TITLE
Ghostery Plus & Insights Promo Modals

### PR DESCRIPTION
**Pull request by wlycdgr**
Everything for the 8.4.4. promo modals release formerly known as 8.4.5, except for the GH-1775 metrics pings ticket (coming separately). That includes:

* Initial version of the Plus promo modal (for display in the Hub and/or Panel) (GH-1776, GH-1777)
* Upgrade version of the Plus promo modal (Panel only) (GH-1813)
* Insights promo modal (Panel only) (GH-1774)
* Logic for deciding if, when, and which modal(s) to display (GH-1773)
* Account-synced promo toggle in Settings->Notifications (GH-1814)

* [X] Have you followed the guidelines in [CONTRIBUTING.md](../CONTRIBUTING.md)?
* [X] Have you checked to ensure there aren't other open [Pull Requests](../pulls) for the same update/change?
* [X] Have you added an explanation of what your changes do?
* [X] Does your submission pass tests?
* [X] Did you lint your code prior to submission?

### CODE REVIEW

File: app/shared-components/PlusPromoModal/PlusPromoModal.scss [hunk 1]
```diff
@@ -0,0 +1,280 @@
+$standard-font-family: Roboto, "Open Sans", "Helvetica Neue", Helvetica, Arial, sans-serif;
```
Comment (christophertino) [inline app/shared-components/PlusPromoModal/PlusPromoModal.scss]: Add a document header / license notice

File: src/background.js [hunk 2]
```diff
@@ -1074,6 +1078,18 @@ function onMessageHandler(request, sender, callback) {
 		});
 		return true;
 	}
+	if (name === 'promoModals.sawPlusPromo') {
+		promoModals.recordPlusPromoSighting();
+		return true;
```
Comment (christophertino) [inline src/background.js]: We should return `false` here since this is synchronous and there is no `callback`

File: src/background.js [hunk 3]
```diff
@@ -1074,6 +1078,18 @@ function onMessageHandler(request, sender, callback) {
...
+	}
+	if (name === 'promoModals.sawInsightsPromo') {
+		promoModals.recordInsightsPromoSighting();
+		return true;
```
Comment (christophertino) [inline src/background.js]: `return false`

File: src/background.js [hunk 4]
```diff
@@ -1074,6 +1078,18 @@ function onMessageHandler(request, sender, callback) {
...
+	}
+	if (name === 'promoModals.turnOffPromos') {
+		promoModals.turnOffPromos();
+		return true;
```
Comment (christophertino) [inline src/background.js]: `return false`

File: app/shared-components/PlusPromoModal/PlusPromoModal.scss (hunk 1 above)
Comment (wlycdgr) [reply app/shared-components/PlusPromoModal/PlusPromoModal.scss]: Done

File: src/background.js (hunk 2 above)
Comment (wlycdgr) [reply src/background.js]: Done

File: src/background.js (hunk 3 above)
Comment (wlycdgr) [reply src/background.js]: Done

File: src/background.js (hunk 4 above)
Comment (wlycdgr) [reply src/background.js]: Done

**Pull request merged**